examples/fc_networks/main.tf

```diff
@@ -14,6 +14,7 @@ data "oneview_scope" "scope_obj" {
 # Creates the fc network with name TestFCNetwork and local name FCNetwork.
 resource "oneview_fc_network" "FCNetwork" {
   name               = "TestFCNetwork"
+  fabric_type        = "FabricAttach"
```

Comment:
its better to provide examples for fabric_type direct attach

Reply:
Done